.pipelines/01-pull-request.yml

```diff
@@ -3,8 +3,8 @@
 resources:
   containers:
   - container: mlops
-    image: public/mlops/tensorflow:latest
-    endpoint: mlopstensorflowacr 
+    image: mlopsmanufacturing/build-agent:latest
```

Comment:
Just try to understand the use case better. The model training itself will be done within AMLS but we could have unit tests related to a specific framework which would run in the CI pipeline. Therefore we need the ML frameworks on the build agent?

Reply:
yes, that's correct.

Reply:
Thanks for bringing the concern up, good to be aware of, I would like to stick to the current "pragmatic" approach of using the same build-agent image until there is a real need to split them and therefore also not add any specific framework "labeling/tagging" to the image name. Please refer to #28 for a more detailed explanation.

Reply:
Great analysis of the problem in #28. I agree we don't need to split the pipeline to run different images right now. However, to implement something like #16 , we'd create a different repo but would make sense to use the same AML workspace. It's just another experiment.  It will have a different agent image in the same ACR.  So this is just to tell the difference between these images.

Reply:
Changed the name to `mlopsmanufacturing/build-agent/tensorflow:latest` 

See PR pipeline [results](https://dev.azure.com/cse-manufacturing/MLOpsManufacturing/_build/results?buildId=376&view=results)